<launch>

<!-- Arm wrench controller -->
  <node pkg="pr2_controller_manager" type="spawner.py" args="$(find arm_life_test)/cartesian_wrench.xml" />

<!-- Test script -->
  <node pkg="arm_life_test" type="random_wrenches.py" args="30 30" />

</launch>
